examples/monitoring_and_alerting/prometheus/alerts/woodpecker_alerts.yml

```diff
@@ -29,3 +29,10 @@ groups:
       severity: warning
     annotations:
       description: 'Errors submitting pre-certificates/certificates to log {{ $labels.uri }}'
+  - alert: OldUnincorporatedCert
+    expr: sum by (uri) (oldest_unincorporated_cert{le_log="true"}) > 7200 # 7200s = 2h
```

Comment:
What is this `le_log` label? It doesn't come from `ct-woodpecker` and I suspect maybe its something SRE is injecting in the Prometheus config based on the source or log URI?

Reply:
Yep, that label is injected in the Prometheus config. Fixed the example. I also chatted with SRE about ways we could tweak our alerting configs so we can maintain the same exact alert configs here as in our actual deployment, making it easier to keep them in sync.

Reply:
>  I also chatted with SRE about ways we could tweak our alerting configs so we can maintain the same exact alert configs here as in our actual deployment, making it easier to keep them in sync.

Is there a ticket that summarizes this discussion and tracks the work?

Reply:
Just filed one, thanks!